index.html

```diff
@@ -188,29 +188,20 @@ <h3>Skills</h3>
 </ul>
 </section>
 <footer class="footer-cont">
-    <h2 class="contact">Contact Me</h2>
+    <h2 class="contact1">Contact Me</h2>
     <p class="footerpara">
         If you have an application you are interested in developing,
          a feature that you need built or a project that needs coding. 
          I’d love to help with it 
     </p>
```

Comment:
- [ ] [ **OPTIONAL** ] Please change the font style and font color according to the Original website.
- Your website
![wedwed](https://user-images.githubusercontent.com/26767383/154147311-35b8e7da-d2b5-4a48-b145-f5a5482232af.PNG)

Reply:
@Deepakdanger  Thank you I have fixed that